index.html

```diff
@@ -1,21 +1,66 @@
-<html>
-<head>
-    <meta charset='utf-8'>
-    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
-    <title>Page Title</title>
-    <link rel='stylesheet' type='text/css' media='screen' href='style.css'>
-</head>
-<body>
-<div class="container">
-    <header></header>
-    <section class="section-1">
-        <h1>I'm Marine Glad to see you</h1>
-        <p>I'm a software Developer! I can help you build aproduct, 
-            feature or website look through some of my work and 
-            exprience! if you like what you se & have aproject you need, 
-            don't hestiate to contact me</p>
-    </section>
-
-</div>
-</body>
-</html>
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,300;0,400;1,300&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,300;0,400;1,300&display=swap"
+    />
+  </head>
+  <body>
+    <div class="container">
+      <main>
+        <nav>
+          <div class="logo">
+            <img
+              src="Asset/My Logo.png"
+              alt="log for My portifolio"
+              class="logo"
+            />
+          </div>
+          <span class="menu-icon">≡</span>
+        </nav>
```

Comment:
- [ ] Great effort working on your project, I would kindly suggest you wrap this section inside of an `header` tag,
   - The container of the section is a `header` tag. Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.

- [ ] Also the `Logo` should be a text wrapped inside an anchor tag `a`. 
     - The logo (if the design has one) is on the left and has a link (`a` tag). This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.

- [ ] Lastly, The Hamburger icon should be wrapped inside the `nav` tag and i would suggest you download the image from Figma rather that using a `span`.
     - The hamburger button is on the right, and has a `nav` tag. The `nav` tag is for semantics reason, and the right position for convention and user experience for mobile devices.

Kindly refactor your header part to be like this. hopefully it will help

```HTML
<header>
   <a href="#">My Logo</a>

   <nav>
       {Hamburger image}
   </nav>
</header>
```